Assets/StreamingAssets.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
This keeps getting generated every time we make builds. I'll create a task to see if it's safe to ignore this on git. But remove in the meantime.

Reply:
I thought I made sure to not include it this time. It really doesn't want to leave us alone